sdk/textanalytics/ai-text-analytics/samples/javascript/package.json

```diff
@@ -0,0 +1,32 @@
+{
+  "name": "azure-ai-text-analytics-samples-js",
+  "private": true,
+  "version": "0.1.0",
+  "description": "Azure Cognitive Services Text Analytics client library samples for JavaScript",
+  "engine": {
+    "node": ">=8.0.0"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/Azure/azure-sdk-for-js.git"
+  },
+  "keywords": [
+    "Azure",
+    "Cognitive Services",
+    "Text Analytics",
+    "Node.js",
+    "JavaScript"
+  ],
+  "author": "Microsoft Corporation",
+  "license": "MIT",
+  "bugs": {
+    "url": "https://github.com/Azure/azure-sdk-for-js/issues"
+  },
+  "homepage": "https://github.com/Azure/azure-sdk-for-js#readme",
```

Comment:
homepage should probably be the text-analytics package readme yeah?

Reply:
Hmm. This is probably wrong in every package actually.